README.md

```diff
@@ -17,4 +17,105 @@ From a cloned copy of this repository, sample data can be loaded into Movilo fro
 </div>
 
 #### Wireframe:
-<img src="https://drive.google.com/uc?export=view&id=17jPsTX7J_bDDp0DaTn-C0XWKU70Gkpin">
\ No newline at end of file
+<img src="https://drive.google.com/uc?export=view&id=17jPsTX7J_bDDp0DaTn-C0XWKU70Gkpin">
+
+# Movilo ![CI/CD](https://github.com/cs481-ekh/f22-viz-kids/workflows/CI/badge.svg)
```

Comment:
Everything above this line can be deleted. It's included in the lines following this one interspersed among the other new additions.